packages/popover/src/popover.css

```diff
@@ -78,3 +78,7 @@
 :host([tip][placement]) #tip {
     height: auto;
 }
+
+:host([picker]) {
```

Comment:
I'm wondering if there's a CSS-only way to approach this without adding an attribute. Do you think we could add a modifier to Popover at the host level?

This is what we have in popover.css now:
```
:host {
    min-width: min-content;
    max-height: 100%;
    max-width: 100%;
    clip-path: none;
}
```

Maybe we create a modifier for width and then in the picker.css file, we can override that and have it cascade down into the popover?